doc/matter/index.rst

```diff
@@ -17,3 +17,5 @@ This documentation is based on the documentation in the `Matter <https://github.
    python_chip_controller_advanced_usage
    android_*
    openthread_*
+   chip_tool_guide
+   access-control-guide
```

Comment:
@kkasperczyk-no , please also update conf.py in doc/matter dir with the path to new files, lines 48 onwards:

    (MATTER_BASE / "docs" / "guides", "android_*.md"),
    (MATTER_BASE / "docs" / "guides", "BUILDING.md"),
    (MATTER_BASE / "docs" / "guides", "chip_tool_guide.md"),
    (MATTER_BASE / "docs" / "guides", "access-control-guide.md"),